### DISCUSSION TITLE
Remux issue and remove toltec if I choose to

### DISCUSSION THREAD

**Discussion by ghost** [toltec-dev/toltec#294] [Support] [answered]
I wanted to install koreader on my rm2 and therefor going with toltec seemed the way to go. I installed rm2fb and remux and there things got a bit wonky. The remux menu never appeared when I swiped up from the bottom. I did see it once or twice but I had the impression that other swipe actions were not as snappy as without remux running. My rm2 is on version 2.5.1.45.  I've tried https://github.com/ddvk/remarkable-autoinstall method to get koreader running but ran into another issue there. But it made me wonder: if I choose to use the ddvk method how do I remove toltec from my device?

**Comment by raisjn** [comment]
thanks for trying out toltec!

> The remux menu never appeared when I swiped up from the bottom

remux appears on swipe up from on left or right side of display. the swipe should be at least 50% of the display, but you can be safe and do a longer swipe at first. 

>  if I choose to use the ddvk method how do I remove toltec from my device?

the main thing is to first remove rm2fb: `opkg remove rm2fb`, then it is safe to remove toltec. toltec mostly lives in /home/root/.entware, but there are some files left in the system (like a systemd mount file for /opt/ or the service files for any launchers still installed in opkg)

**Reply by ghost** [reply]
Thanks for the quick reply... I will give it another try.

**Reply by ghost** [reply]
I tried it again, and indeed making the swipe longer does the trick. It doesn't always work and also depents on the pressure.  It's not consistent, sometimes I can do a light swipe and sometimes I have to put some pressure on it. Sometimes the left of the screen works better then the right.  Since everthing is in full development I thought it might be useful to mention.

**Reply by raisjn** [reply]
try tapping screen with finger before doing the gesture. in later version of remux, it is better but am waiting on merge into toltec: #286

**Reply by ghost** [reply]
I did the 3 finger tap and that worked much better. But I kept having trouble with the swiping withing xochitl. Since this is still in development perhaps its good to know.  After a restart everthing runs fine, but after a short while using xochitl it doesn't register the swipes accurately ( this doesn't happen within koreader ).  So turning a page doesn't always register and I sometimes have to apply more pressure. Often I had to try it several times before the page would turn.  As soon as I stopped remux, restarted xochitl this does not occur.  For now I use ddvk koreader autoinstall script. I loose the multitasking, but don't experience swipe issues.  His touchinjector doesn't seem to interfer with xochitl. If only his touchinjector would bring up the remux menu ;-)

**Reply by raisjn** [reply]
thanks for investigating, this gives me some solid leads. you can have touchinjector bring up remux by having the swipeup.sh (look in repo for directions) run `echo show > /run/remux.api`

**Comment by danshick** [comment, marked as answer]
I created #295 to address this as well. Uninstallation should be automated.
